presto-main/src/main/java/com/facebook/presto/server/KerberosInternalCommunicationConfig.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.server;
+
+import io.airlift.configuration.Config;
+
+import javax.validation.constraints.NotNull;
+
+import java.io.File;
+
+public class KerberosInternalCommunicationConfig
+{
+    private String kerberosPrincipal;
+    private String kerberosServiceName;
+    private File kerberosKeytab;
+    private File kerberosConfig;
+    private boolean kerberosUseCanonicalHostname = true;
```

Comment:
After you remove all the other properties, this one is gonna be the only one left. I would just move it to the `InternalCommunicationConfig `